iidm-import-export-api/src/main/java/eu/itesla_project/iidm/datasource/GenericReadOnlyDataSource.java

```diff
@@ -20,7 +20,8 @@
     public GenericReadOnlyDataSource(Path directory, String baseName, DataSourceObserver observer) {
         dataSources = new DataSource[] { new FileDataSource(directory, baseName, observer),
                                          new GzFileDataSource(directory, baseName, observer),
-                                         new ZipFileDataSource(directory, baseName + ".zip", baseName, observer) };
+                                         new ZipFileDataSource(directory, baseName + ".zip", baseName, observer),
+                                         new Bzip2FileDataSource(directory, baseName, observer) };
```

Comment:
I don't know if we need to sort the datasource (performance issue, used to scan the case repository... or something else). If sorting the datasources is relevant, maybe "Bzip2FileDataSource" should be in first or second position ?